src/forms/input/index.tsx

```diff
@@ -18,10 +18,12 @@ type InputProps = {
     disabled?: boolean
     helperText?: ReactNode
     label?: string
+    type?: string
```

Comment:
I think 'type' prop is already present at line 17 in same file.

Reply:
removed the duplicate